docs/DeveloperGuide.md

```diff
@@ -201,6 +201,65 @@ This section describes some noteworthy details on how certain features are imple
 
 --------------------------------------------------------------------------------------------------------------------
 
+
+### Add patient feature
+
+#### Implementation
+
+The add patient operation is facilitated by `AddCommandParser`. `AddCommandParser` parses the user input string
+and creates the `AddCommand` to be executed by the `LogicManager`. `AddCommand` extends `Command` and implements the
+`Command#execute` method.
+
+Given below is an example usage scenario and how the add patient operation is handled in A&E.
+
+Step 1. Assuming the application has been launched, the user enters the required fields followed by any optional fields.
+
+For adding only required fields, the user enters `add n/John Tan i/t7654321j`, which is to add
+a patient with `Name = John Tan` and `IcNumber = t1234567j`,
+
+For adding additional optional fields, the user enters `add n/John Tan i/t7654321j p/90909090`, which is to add the
+optional field `Phone = 90909090`.
+
+The `AddCommandParser#createPatient` method creates the patient with the relevant fields and fills all optional fields
+with default values otherwise.
+
+
+Step 2. `LogicManager#execute` would first invoke `AddressBookParser#parseCommand` which splits the
+command word `view` and the argument `i/t1234567j`. After splitting, `AddressBookParser#parseCommand` would identify
```

Comment:
Should this be `add` instead of `view`? Same for steps 3, 4 & 5.

Reply:
yup have yet to update it, will do so in next pr